content/docs/1.5/scalers/cron.md

```diff
@@ -27,6 +27,15 @@ triggers:
 - `timezone` is one of the acceptable values from the IANA Time Zone Database. The list of timezones can be found in: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones. Required.
 - `start` is the cron expression indicating the start of the cron schedule. Required.
 - `end` is the cron expression indicating the end of the cron schedule. Required.
+
+> **Notice:**
+> - **start and end should not be same.** For example:
+> ```yaml
+> start: 30 * * * *
+> end: 30 * * * *  # Invalid input
+>```
+> - The above schedule is invalid.
```

Comment:
What about this?

```suggestion
> **Notice:**
> **Start and end should not be same.**
> 
> For example, the following schedule is not valid:
> ```yaml
> start: 30 * * * *
> end: 30 * * * *
>```
```

Reply:
Sure @tomkerkhove I'll make the change. Any other suggestion?